src/map/entities/mobentity.cpp

```diff
@@ -337,7 +337,7 @@ uint8 CMobEntity::TPUseChance()
 
     if (health.tp == 3000 || (GetHPP() <= 25 && health.tp >= 1000))
     {
-        return 100;
+        return 10000;
     }
 
     return (uint8)getMobMod(MOBMOD_TP_USE_CHANCE);
```

Comment:
Not really critical on this occasion, but could change to uint16 to match the return type.